sfge/src/main/java/com/salesforce/graph/source/supplier/GlobalMethodSupplier.java

```diff
@@ -33,10 +33,11 @@ public List<MethodVertex> getVertices(GraphTraversalSource g, List<String> targe
                                         out(Schema.CHILD)
                                                 .hasLabel(ASTConstants.NodeType.MODIFIER_NODE)
                                                 .has(Schema.GLOBAL, true),
-                                        // Ignore any standard methods, otherwise will get a ton
-                                        // of
-                                        // extra results.
-                                        __.not(__.has(Schema.IS_STANDARD, true)))));
+                                        // Ignore any standard and implicit methods, otherwise will
+                                        // get a ton
+                                        // of extra results.
+                                        __.not(__.has(Schema.IS_STANDARD, true)),
+                                        __.not(__.has(Schema.IS_IMPLICIT, true)))));
```

Comment:
Line 28 of this file was filtering out methods that had no block statements. Since we're synthesizing block statements for implicit constructors now, we need to exclude them in a different way. Hence this line.

Reply:
I do not think i follow this

Reply:
This method is querying the graph for every vertex that represents a `global`-scoped method. At lines 28-31, we're filtering out methods without a `BlockStatement` child. Before this PR, the auto-generated default constructors had no `BlockStatement` and were thus caught by that exclusion. Since this PR creates synthetic `BlockStatement`s for those constructors, the exclusion is no longer sufficient and this new one has to be added.